Documentation/ceph-upgrade.md

```diff
@@ -63,6 +63,8 @@ need these features now, you can skip this step for the patch upgrade.
 
 ## Enabling the CSI v2.0 driver
 
+> If you are deploying the rook-operator with the helm chart you only need to specify the related images in your [values.yaml](https://github.com/rook/rook/blob/release-1.2/cluster/charts/rook-ceph/values.yaml#L91). The RBAC settings will be created for you by the chart.
```

Comment:
do we need to use release-1.2 branch or master here?

Reply:
I just realized I created the branch against master when initially I wanted to point towards the 1.2 branch. Maybe I should change to master here and then create PR for the other releases?

Reply:
Actually these links are easily broken since the line numbers change over time. What if we didn't link the values.yaml? Is it self-explanatory enough for helm users?

Reply:
Or maybe it would make more sense if I do them against the releases as the values.yaml can change in the future.